src/media/dto/mediaUpload.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsIn, IsUUID } from 'class-validator';
+import { entityType } from 'src/shared/enums/entityType.enum';
+
+export class MediaUploadDTO {
+  @IsIn([entityType.OPTION, entityType.POST, entityType.OPTION_GROUP])
+  entity_type: string;
+
+  @IsUUID(4, { message: 'entity id is not a valid uuid' })
+  entity_id: string;
```

Comment:
I don't understand what does the word `entity` refers to?
Which entity?

Shouldn't it be something else like `post` ?

Reply:
entity here refers to (post | optionsGroup | option) entities. So what would you suggest to name it.